server/src/main/java/org/elasticsearch/index/engine/CombinedDeletionPolicy.java

```diff
@@ -228,15 +228,15 @@ protected IndexCommit wrapCommit(IndexCommit indexCommit) {
      */
     synchronized boolean releaseCommit(final IndexCommit snapshotCommit) {
         final IndexCommit releasingCommit = ((SnapshotIndexCommit) snapshotCommit).getIndexCommit();
-        assert snapshottedCommits.containsKey(releasingCommit)
+        assert acquiredIndexCommits.containsKey(releasingCommit)
             : "Release non-snapshotted commit;"
```

Comment:
nit Can we also rename here snapshotted -> acquired? similar the method argument `snapshotCommit` to `acquiredCommit`?

And also rename `snapshot` argument of `InternalEngine#releaseIndexCommit`?

Reply:
Renamed this and a few more related places 5735ee81322b35daa6bb9e83fc1092d2d61225bb
I suspect we could have more naming consolidation for consistency as follow-ups. For example, we have this [SnapshotIndexCommit](https://github.com/elastic/elasticsearch/blob/10139a30ed1a4caa86b543f14b8417594306daf5/server/src/main/java/org/elasticsearch/index/engine/CombinedDeletionPolicy.java#L322) class that is used internally by CombinedDeletionPolicy.